modules/osdk-installing-cli.adoc

```diff
@@ -34,6 +22,52 @@ endif::[]
 
 .Procedure
 
+. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/clients/[OpenShift mirror site] and choose the Operator SDK folder for your operating system and architecture.
```

Comment:
This procedure is somewhat placeholder until the final download location / file name of the client is confirmed.

Reply:
Confirmed now that the two archs are Linux and MacOS, so updating this placeholder line accordingly.